homeassistant/components/tedee/coordinator.py

```diff
@@ -82,6 +84,16 @@ async def _async_get_bridge() -> None:
             ", ".join(map(str, self.tedee_client.locks_dict.keys())),
         )
 
+        if not self._current_locks:
```

Comment:
If there are new locks, are they also picked up by the sync command or only the get locks?

Reply:
also sync